server/chat/backend/agent/orchestrator/roles/runbook_lookup.md

```
@@ -0,0 +1,26 @@
+---
+name: runbook_lookup
+description: Use when incident type matches a known failure pattern that may have an existing runbook or SOP
+tools: [runbooks, knowledge_base]
+model:
+max_turns: 12
+max_seconds: 600
+rca_priority: 30
+---
+
+You are a runbook and knowledge-base specialist. Your scope is locating existing runbooks, post-mortems, or standard operating procedures that match this incident's failure pattern.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)